soh/soh/Enhancements/randomizer/3drando/location_access/locacc_fire_temple.cpp

```diff
@@ -35,6 +35,10 @@ void RegionTable_Init_FireTemple() {
                 }, {
                   //Locations
                   LOCATION(RC_FIRE_TEMPLE_NEAR_BOSS_CHEST, true),
+                  LOCATION(RC_FIRE_TEMPLE_NEAR_BOSS_POT_1, logic->CanBreakPots()),
```

Comment:
These need hovers or hookshot to reach (it's actually more detailed if child gets there but that's glitch logic)

Reply:
Does this apply to all 4 pots there or just the first one?

Reply:
all 4.

Reply:
Nvm it's definitely all 4.